charts/csm-installer/values.yaml

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Do we have a table in documentation that outlines the helm values?

Reply:
We detail how to update some of the required values (jwtKey, cipherKey, admin credentials) in our common doc portal, but we don't have a full listing. I'll make a note to add a full table with all of the values on that common doc site.